lib/ccavenue-sdk.rb

```diff
@@ -363,11 +363,12 @@ def order_status_updated_at
     end
 
     ### merchant credentials validation
-    VALIDATION_SUCCESS_MSG = 'Providing Reference_No/Order No is mandatory'
+    VALIDATION_SUCCESS_MSGS = ['Providing Reference_No/Order No is mandatory',
```

Comment:
This ideally should be a config setting so if this changes again we don't have to do a patch deploy. We can do this in the next release so won't hold for this, but will create JIRA so it gets done for next release.